### DISCUSSION TITLE
How to get mesh syncing & syncback working?

### DISCUSSION THREAD

**Discussion by pxky** [argon-rbx/argon#138] [Q&A]
When I try to sync a .rbxm with a meshpart argon throws an error:

![Code_ABaiYWeTSn](https://github.com/user-attachments/assets/6458a0c1-7f5e-4cba-9261-3d89d4313bba)

However when I run vscode in Administrator mode the above goes away but I then get this in studio:

![RobloxStudioBeta_YqoSozraZv](https://github.com/user-attachments/assets/1ee1d25a-114b-4dca-8cbf-309b779febe5)

I also cant figure out how to get syncback working, are there any guides for this? The link in docs point to Argon Plugin settings where I believe I have them set up correctly...

**Comment by DervexDev** [comment]
See #158
